### PR TITLE
Added my blog

### PR DESCRIPTION
**Pull request by pmsbir**
Spot added the link of my blog as a part of FOSSASIA task.

Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Comment by abishekvashok** [comment]
@pmsbir please state what have you done here. Thanks. Also pls resd the readme before continuing.

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@pmsbir  I think you want to add a blog, Please `README.md` on how to add a blog on site . 👍

**Review by nikit19 — COMMENTED** [review]
Try writing a meaningful pull request title.
So that it becomes easier to understand what you are trying to do.

**Comment by pmsbir** [comment]
I have added my blog but am unaware of how to put a pull request...what should I do..?
This is my GCI task.

File: _data/blogs.yml [hunk 1]
```diff
@@ -191,4 +191,10 @@
 - url: https://saarthakchaturvedi05.wordpress.com/2017/12/24/interview-with-long-term-open-source-contributors/
   title: Interview with Two Long-Term Open Source Contributors
   img: saarthakchaturvediinterview.jpg
-  author_name: Saarthak Chaturvedi
\ No newline at end of file
+  author_name: Saarthak Chaturvedi
+  
+- url : https://limitless497584285.wordpress.com/2017/12/27/coding-my-joy/
+  title : CODING- MY JOY
+  img : https://photos.app.goo.gl/Ztih0n6GZrS1NyY43
```
Comment (nikit19) [inline _data/blogs.yml:198]: After img : you have to write the name of the image that you have uploaded.

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please upload the image in the correct folder ie img/blog and resolve conflicts

**Comment by akshatnitd** [comment]
Closing this since the student has opened up another PR #570

**Pull request closed without merging**